### ISSUE TITLE
The train and val accuracy suddenly drop to 0.0X

### ISSUE THREAD

**Issue by dami2014** [IcewineChen/mxnet-stacked_hourglass#2]
Hi Icewine,
     I have used you shared code to train my face alignment project with 68 points, after 10 epochs with almost 56% accuracy, the train and val accuracy suddenly drop to 0.0X.  This problem can be reproduced(i have tied more 5 times) in mxnet. Same problem also occurs in other architecture(Pytorch), but only val accuracy drops(https://github.com/bearpaw/pytorch-pose/issues/33).
Below is log:
Epoch	LR	Train Loss	Val Loss	Train Acc	Val Acc	
1.000000	0.000100	0.003555	0.000221		0.003957	0.004911	
2.000000	0.000100	0.000435	0.000214		0.005413	        0.005837	
3.000000	0.000100	0.000422	0.000211		0.005972	0.004650	
4.000000	0.000100	0.000389	0.000185  	0.019450	0.036107	
5.000000	0.000100	0.000357	0.000177	 	0.066919	0.100733	
6.000000	0.000100	0.000326	0.000156	 	0.160107	        0.192923	
7.000000	0.000100	0.000301	0.000144  	0.225927	0.251442	
8.000000	0.000100	0.000286	0.000137	 	0.310324	        0.386344	
9.000000	0.000100	0.000275	0.000134  	0.440678	0.513889	
10.000000	0.000050	0.000267	0.000129	 	0.559680	0.614324	
11.000000	0.000050	0.000266	0.000132	 	0.608763	0.599997	
12.000000	0.000050	0.000330	0.000165	 	0.548631	0.341394	
13.000000	0.000050	0.000364	0.000167	 	0.472620	0.434283	
14.000000	0.000050	0.000390	0.000313	 	0.313622	        0.063162	
15.000000	0.000050	0.000429	0.000213	 	0.044952	0.010647	
Version: mxnet 1.3.0

**Comment by ruyiweicas** [comment]
@dami2014  Can you provide specific test procedures?Let me repeat that

**Comment by ruyiweicas** [comment]
@dami2014  可以给我一个你的联系方式吗

**Comment by IcewineChen** [comment]
@dami2014 Sorry for answering you so late. I just test my mxnet-version network on mpii human pose dataset. Maybe the dropout problem caused by data? I guess.

**Comment by dami2014** [comment]
@ruyiweicas I will update the train code, after the festival. liulei2140@gamil.com

**Comment by dami2014** [comment]
@IcewineChen The data i trained successfuly have trained on other face alignment network in the framework of mxnet. I just modify the fist convlution kernel size 7*7 to 3*3 to avoid feature map scale down.The input image size is 112*112(not 256*256). I will check the tain data carefully.

**Comment by IcewineChen** [comment]
@dami2014 Thanks for your reply ☺
